doc/getting-started/install.md

```diff
@@ -42,7 +42,7 @@ For Debian/Ubuntu, use the following instead:
 
 ```bash
 sudo apt-get update -y
-sudo apt-get install automake build-essential pkg-config libffi-dev libgmp-dev libssl-dev libtinfo-dev libsystemd-dev zlib1g-dev make g++ tmux git jq wget libncursesw5 libtool autoconf liblmdb-dev -y
+sudo apt-get install automake build-essential pkg-config libffi-dev libgmp-dev libssl-dev libtinfo-dev libsystemd-dev zlib1g-dev make g++ tmux git jq wget libncursesw5 libtool autoconf liblmdb-dev llvm libnuma-dev -y
```

Comment:
What are these extra dependencies for?

Reply:
Those two are not hard dependencies and should not be required for a vanilla GHC installation.  Users may have installed a special version of GHC or other build tool that imposes this dependency.  I think for now list them as optional dependencies that may be required depending on the user's installation.